InfiniteRecharge/src/main/java/frc/robot/subsystem/scoring/intake/IntakeSubsystem.java

```diff
@@ -33,14 +36,14 @@ public IntakeSubsystem(Config config) {
     @Override
     public void initialize() {
         super.initialize();
-        if (config.intake.intakePivotEnabled){

```

Comment:
We still want this. We have an intake pivot configured for Shawty, but not a robot like Junior. It's not a constant.